service/src/main/resources/application.yaml

```diff
@@ -247,16 +247,6 @@ security:
   anonymousPaths:
   authorisedServices:
 
```

Comment:
There is another place in that file with the feature config.

Reply:
you are right, removed it now.